File: src/main/scala/ch11/Ex1.java
package ch11;

public class Ex1 {
    // This is Java
    boolean isEqual(int x, int y) {
        return x == y;
    }

    public void go() {
        System.out.println(isEqual(421, 421));
    }

    public static void main(String[] args) {
        Ex1 ex1 = new Ex1();
        ex1.go();
    }
}

File: src/main/scala/ch03/Ex1.scala
package ch03

object Ex1 {
  def main(args: Array[String]) {
    val big = new java.math.BigInteger("12345")
    println("big [" + (big) + "]")

    val greetStrings = new Array[String](3)

    greetStrings(0) = "Hello"
    greetStrings(1) = ", "
    greetStrings(2) = "world!\n"

    for (i <- 0 to 2)
      print(greetStrings(i))
  }
}
